.husky/pre-commit

```diff
@@ -1,3 +1,5 @@
+yarn typegen
```

Comment:
added this here so we don't have to remember to make the types up to date if were ever making sanity changes